go/fixchain/fix_error.go

```diff
@@ -39,6 +44,8 @@ func (e FixError) TypeString() string {
 		return "CannotFetchURL"
 	case FixFailed:
 		return "FixFailed"
+	case PostFailed:
```

Comment:
The idea is you modify the code to explain this (presumably in the comment where the error is defined).